beacon_node/eth1/src/service.rs

```diff
@@ -255,11 +137,11 @@ async fn get_remote_head_and_new_block_ranges(
             "last_seen_block_unix_timestamp" => remote_head_block.timestamp,
             "action" => "trying fallback"
         );
-        return Err(SingleEndpointError::EndpointError(EndpointError::FarBehind));
+        return Err(Error::EndpointError(EndpointError::FarBehind));
     }
 
     let handle_remote_not_synced = |e| {
-        if let SingleEndpointError::RemoteNotSynced { .. } = e {
+        if let Error::RemoteNotSynced { .. } = e {
             warn!(
```

Comment:
there's an `"action"` key in this warn log that says `"trying fallbacks"`. we should delete it